### DISCUSSION TITLE
PersistentWindows.exe - Assert Failure

### DISCUSSION THREAD

**Discussion by AuricMists** [kangyu-california/PersistentWindows#366] [General]
Hi Kangyu,
FYI, I just got this assert failure on booting my PC. It didn't seem to affect things as it still worked.
Regards, Auric
![Screenshot_160](https://github.com/user-attachments/assets/e4eb22b8-a487-4d04-8f6c-bcac95943107)

**Comment by kangyu-california** [comment]
Thanks for finding out.
I am not sure why line 1631 is causing KeyNotFound assertion.
Please try latest patch
[PersistentWindows5.59_patch6.zip](https://github.com/user-attachments/files/18088699/PersistentWindows5.59_patch6.zip)

If the issue persists, please be relaxed, as the assertion failure only happens when rebooting PC, there is no negative impact on user experience.

**Comment by AuricMists** [comment]
It's not a problem, I just thought you might like to know.
As a retired software developer myself, I know how frustrating it can be to
try to get the client to record errors. 😶😄

Cheers and have a good Solstice and all that ...
